src/Agent/NewRelic/Agent/Core/Config/Configuration.xsd

```diff
@@ -347,19 +347,6 @@
                             </xs:annotation>
                         </xs:attribute>
 
-                        <xs:attribute name="fileLockingModel">
```

Comment:
Since this is experimental, is it safe to remove with a regular release, or do we need to wait for a major release?

Reply:
Maybe we could just mark it as deprecated? It shouldn't be necessary at all with Serilog, and if it is, that would probably be a bug.

Reply:
Verified that the Agent continues to start and run correctly even with this attribute removed from the XSD and generated .cs file.